aws-ecs/telegraf.conf

```diff
@@ -54,7 +54,7 @@
 
   ## ecs labels to include and exclude as tags.  Globs accepted.
   ## Note that an empty array for both will include all labels as tags
-  ecs_label_include = [ "com.amazonaws.ecs.*" ]
```

Comment:
How many labels are we getting by default and how many with this include tag?

Reply:
Below are the labels available at the filter
•	com.amazonaws.ecs.cluster
•	com.amazonaws.ecs.container-name
•	com.amazonaws.ecs.task-definition-family
•	com.amazonaws.ecs.task-arn
•	com.amazonaws.ecs.task-definition-version
•	cluster
•	family
•	id
•	name
•	network
•	revision
•	task_arn
•	source
•	tag